cmd/kubeadm/app/kubeadm.go

```diff
@@ -20,15 +20,19 @@ import (
 	"flag"
 	"os"
 
-	_ "github.com/golang/glog"
+	glog "github.com/golang/glog"
```

Comment:
Just a tiny nit, you can drop the glog alias here and leave this line as `"github.com/golang/glog"`